query-api/src/main/java/com/brandwatch/minibcr/queryapi/repository/UserQueryRepository.java

```diff
@@ -0,0 +1,24 @@
+package com.brandwatch.minibcr.queryapi.repository;
+
+import java.util.List;
+
+import org.springframework.cache.annotation.CacheEvict;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import reactor.util.annotation.NonNull;
+
+import com.brandwatch.minibcr.queryapi.entity.UserQuery;
+
+@Repository
+public interface UserQueryRepository extends JpaRepository<UserQuery, Long> {
```

Comment:
Just wanted to point out that at BW we typically don't use JPA as I imagine Houssam will have pointed out to you. So you might want to consider looking into `NamedParameterJdbcTemplate` which is typically used. But if you want to continue using JPA here, you can.